submit_async.go

```diff
@@ -0,0 +1,79 @@
+package gosnowflake
+
+// this is a sigma only file
+// when rebasing you can ignore everthing in this file as long as this
+// file makes it into master
+
+import (
+	"context"
+	"database/sql/driver"
+	"strconv"
+)
+
+type isComplete bool
```

Comment:
I'm not sure that defining `isComplete` as a type buys us much, other than slightly better documentation. But I don't think this would be necessary if, instead of returning multiple values, we return a struct that has `IsComplete` and `QueryId` as fields.

Reply:
oh good idea I will do that